client/src/main/scala/drt/client/components/ForecastFileUploadPage.scala

```diff
@@ -0,0 +1,89 @@
+package drt.client.components
+
+import diode.data.Pot
+import drt.client.actions.Actions.{ForecastFileUploadAction, ResetFileUpload}
+import drt.client.modules.GoogleEventTracker
+import drt.client.services.SPACircuit
+import japgolly.scalajs.react.component.Scala.Component
+import japgolly.scalajs.react.vdom.html_<^._
+import japgolly.scalajs.react.{Callback, CtorType, ScalaComponent, _}
+import org.scalajs.dom
+import org.scalajs.dom.raw.{FormData, HTMLFormElement}
+
+
+case class FileUploadState(state: String, message: String)
+
+case class FileUploadStateModel(fileUploadState: Pot[FileUploadState])
+
+object ForecastFileUploadPage {
+
+  case class Props()
+
+  val component = ScalaComponent.builder[Props]("ForecastFileUpload")
+    .render_P(_ =>
+      <.div(^.className := "fileUpload",
+        <.h3("Forecast Feed File Upload"),
+        <.form(<.input(^.`type` := "file", ^.name := "filename"),
+          <.input(^.`type` := "button", ^.value := "send", ^.onClick ==> onSubmit)),
+            <.br(),
+        ResultFileUploadComponent(),
```

Comment:
FileUploadResultComponent would be a better name I think.

Reply:
👍